dlog/commitment/src/commitment.rs

```diff
@@ -660,62 +665,50 @@ where
         let mut g = self.g.clone();
         g.extend(vec![G::zero(); padding]);
 
-        let mut combined_polynomial_terms = vec![];
-
-        // scale the polynoms in accumulator shifted, if bounded, to the end of SRS
         let (p, blinding_factor) = {
-            let mut p = DensePolynomial::<Fr<G>>::zero();
-
-            let mut omega = Fr::<G>::zero();
+            let start = std::time::Instant::now();
```

Comment:
btw it feels weird to have these timers in prod code. Wouldn't it be better to extract this logic in its own function and then write a bench for it?

Reply:
Hm I think I have deleted this in a later commit